apollo-gradle-plugin/src/test/kotlin/com/apollographql/apollo3/gradle/test/KotlinPluginVersionTests.kt

```diff
@@ -29,12 +29,26 @@ class KotlinPluginVersionTests {
   }
 
   @Test
-  fun kotlin14Succeeds() {
+  fun kotlin14Fails() {
     TestUtils.withTestProject("kotlin-plugin-version") { dir ->
       dir.setKotlinPluginVersion("1.4.32")
-      val result = TestUtils.executeTask("help", dir)
+      try {
+        TestUtils.executeTask("build", dir)
+        fail("An exception was expected")
+      } catch (e: UnexpectedBuildFailure) {
+        Truth.assertThat(e.message).contains("Incompatible classes were found in dependencies. Remove them from the classpath or use '-Xskip-metadata-version-check' to suppress errors")
```

Comment:
Do we really need this test? It's interesting to know but it downloads yet another version of Kotlin, spawns yet another GradleRunner instance, etc... So I'd be happy with a comment instead :)

Reply:
Oh yea, definitely 👍